### PR TITLE
Feat: Added image component

### PR DESCRIPTION
**Pull request by Manpreet-hub**
- Added responsive image.
- Added round image.

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
looks good to merge

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.img {
+    max-width: 34rem;
```
Comment (Prabhav-Sharma) [inline image/image.css:2]: Why have you given this specific height and width?

Comment (Manpreet-hub) [reply image/image.css:2]: If I will not give then it will cover all the available width & height space. By giving this I have defined a particular range.

**Review by Bharatrathod-81 — APPROVED** [review]
good enough for merge

File: image/image.css (hunk 1 above)
Comment (tewarimohit) [reply image/image.css:2]: Is it not hampering image's responsiveness?

**Review by tewarimohit — COMMENTED** [review]
Just go through comments once in css file..

File: image/image.css (hunk 1 above)
Comment (Prabhav-Sharma) [reply image/image.css:2]: Okay

**Pull request merged**